static/scss/partials/dashboard.scss

```diff
@@ -204,12 +204,16 @@
 
 .c-alias-name {
     margin-left: $spacing-xs;
-    max-width: $content-sm;
     height: 100%;
     flex-wrap: nowrap;
+    max-width: 100%;
 
-    .additional-notes, .relay-address {
+    .additional-notes {
         text-overflow: ellipsis;
+        flex-direction: row;
+        position: relative;
+        margin: auto auto 0 0;
+        max-width: $content-xs;
```

Comment:
Question: Should this be `$content-sm` to match what was on `.c-alias-name`? I'm fine with either!